src/sec_edgar_bulker/__init__.py

```
@@ -0,0 +1,23 @@
+
+__version__ = "0.1.0"
+
+__all__ = [
+    'EdgarDownloader',
+    'IdxDownloader',
+    'HeaderGenerator',
+    'ProxyManager',
+    'Config',
+    'setup_logging',
+    'validate_config'
+] 
```

Ruff (0.8.2)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)